### PR TITLE
chore: replace `BlockStore` and `StateStore` with `BlockDb` and `StateDb`

### LINKED ISSUE #686: Rename `BlockStore` and `StateStore`
We should stop naming everything `Store`. For these cases I suggest the names `BlockDb` and `StateDb`

### PR DESCRIPTION
**Pull request by manojkgorle**
**Motivation**

**Description**

Closes #686

### CODE REVIEW

**Comment by MegaRedHand** [comment]
The CI is failing due to unordered import lists in some places.

**Comment by manojkgorle** [comment]
I have fixed them now. let me know if there are any changes to make.

**Comment by MegaRedHand** [comment]
Please sync with main

**Comment by manojkgorle** [comment]
Synced with main.

File: lib/lambda_ethereum_consensus/beacon/beacon_node.ex [hunk 1]
```diff
@@ -78,6 +80,9 @@ defmodule LambdaEthereumConsensus.Beacon.BeaconNode do
     Supervisor.init(children, strategy: :one_for_all)
   end
 
+  defp fetch_anchor_block(%Types.BeaconState{} = anchor_state) do
+    block_root = Misc.get_latest_block_hash(anchor_state)
+    BlockDb.get_block(block_root)
```
Comment (MegaRedHand) [inline lib/lambda_ethereum_consensus/beacon/beacon_node.ex:85]:
```suggestion
```

**Review by MegaRedHand — CHANGES_REQUESTED** [review]
There is an old code fragment that should be removed. Also, don't forget you can run `make lint` locally to check the `CI / lint` job's output.

**Comment by manojkgorle** [comment]
updated to main.

**Review by MegaRedHand — APPROVED** [review]
Thanks for your contribution! 😄

**Comment by MegaRedHand** [comment]
@all-contributors please add @manojkgorle for code

**Pull request merged**